src/source-nfq.c

```diff
@@ -407,6 +408,8 @@ static int NFQSetupPkt (Packet *p, struct nfq_q_handle *qh, void *data)
     char *pktdata;
     struct nfqnl_msg_packet_hdr *ph;
 
+    p->ReleasePacket = NFQReleasePacket;
```

Comment:
in the repeat mode block below we can already set a verdict and return. I think in this case we'd still call NFQReleasePacket, but we probably didn't set p->nfq_v.verdicted, so we'd verdict again?

Reply:
Should the repeat mode block use `NFQSetVerdict` instead of doing it inline?

Reply:
I've updated this to address the issue you raised.